### PR TITLE
Added messages for DBV2

### PR DESCRIPTION
**Pull request by ItsTimmy**
## Description

To support Duckiebot V2, new messages must be added. This change only adds messages, and does not change any existing messages. Therefore, it does not break anything, or have any dependencies. To see where these messages are used, see the branch `merlin-merge` in the following repositories:

- [dt-core-dbv2](https://github.com/duckietown/dt-core-dbv2)
- [dt-duckiebot-interface-dbv2](https://github.com/duckietown/dt-duckiebot-interface-dbv2)
- [dt-car-interface-dbv2](https://github.com/duckietown/dt-car-interface-dbv2)

These messages were originally changed by @hosnerm in [this branch](https://github.com/hosnerm/Software/tree/devel-dbv2-LQRI-encoder-18). 

## Type of change

Please mark the relevant options with X:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change is constrained only to one package, any interfaces to nodes from other packages are maintained
- [ ] This change is constrained only to packages in this repository, any interfaces to nodes from other repositories are maintained
- [ ] This change required a documentation update
- [ ] This change is only a documentation update
- [ ] This change restructures the repository (at level higher than package level)
- [ ] This change requires a change in the Duckiebook (if yes, please describe and, if possible, make a pull request for an edit in the book and link it here)

## Tests

Please describe all tests ran and their outcome. It is highly recommended that you at least do the tests listed by default, even if you only changed the documentation.

- [X] Made the Duckiebot move with keyboard control
- [X] Visually checked the published camera stream
- [X] Verified that the lane following demo works
- [X] Verified that the indefinite navigation demo works

## Style compliance
Please make sure that all your changes conform to the Duckietown (code style)[link] and code (documentation style)[link] guidelines:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I checked that this pull request is compliant with the Duckietown code style
- [X] I checked that this pull request is compliant with the Duckietown code documentation style

## Next steps
Please describe if you have any questions, comments, or doubts towards the Duckietown software guardians regarding implementation, code style, running tests, documentation, etc.

## FOR THE REVIEWERS
__IMPORTANT:__ DO NOT CHANGE THIS SECTION!

- [ ] Functionality (is the change necessary and is it implemented in the most appropriate way): @liampaull or @afdaniele
- [ ] Integration testing (verify the tests and run additional if necessary): @?
- [ ] Code style compliance: @gibernas
- [ ] Documentation compliance: @aleksandarpetrov

_To the reviewers:_ If you deem necessary, request code edits, more tests, documentation update, or additional reviewing. __Do not review your own code!__

_PR Template, last edit 23 Aug 2019_

### CODE REVIEW

**Comment by surirohit** [comment]
@honserm 

Why is there a stamped message, and a non-stamped message for wheels commands?

**Comment by ItsTimmy** [comment]
@afdaniele @AmaurX

File: packages/duckietown_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -69,6 +71,8 @@ add_message_files(
   Vsample.msg
   WheelsCmd.msg
   WheelsCmdStamped.msg
+  WheelsCmdDBV2.msg
```
Comment (AmaurX) [inline packages/duckietown_msgs/CMakeLists.txt]: To be removed (according to discussion)

File: packages/duckietown_msgs/msg/WheelsCmdDBV2.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+float32 gamma           #"vel_left" changed to "gamma", RFMH_2019_02_26
+float32 vel_wheel       #"vel_right" changed to "vel_wheel", RFMH_2019_02_26
+float32 trim            #included "trim" to be accessible in the wheels_driver_node as well, RFMH_2019_04_01
```
Comment (AmaurX) [inline packages/duckietown_msgs/msg/WheelsCmdDBV2.msg]: to be removed

**Comment by surirohit** [comment]
@afdaniele 
Approved for merging

**Pull request merged**